api/v1beta2/foundationdb_database_configuration.go

```diff
@@ -76,9 +77,11 @@ type Region struct {
 // ExcludedServers represents the excluded servers in the database configuration
 type ExcludedServers struct {
 	// The Address of the excluded server.
+	// +kubebuilder:validation:MaxLength=20
 	Address string `json:"address,omitempty"`
 
 	// The Locality of the excluded server.
+	// +kubebuilder:validation:MaxLength=100
```

Comment:
```suggestion
	// +kubebuilder:validation:MaxLength=200
```

W should add some space for longer localities or long values.